### ISSUE TITLE
Add documentation to each function and data type of function variables

### ISSUE THREAD

**Issue by Rene36** [mamin27/ecomet_i2c_raspberry_tools#1]
For somebody reading everything for the first time it quiet hard to follow.
Therefor, I recommend to add a short description to each function with the data type of the input and return variables and what they mean.
```
def function-a(self, a: int, b: float) -> bool:
    """
    Function description here

    :param a: int, random integer
    :param b: str, random float
    :return: bool, true if 1th parameter is bigger than the 2nd
    """
    if a > b:
        return True
    else:
        return False
```

**Comment by mamin27** [comment]
Thanks for advice Rene. I will update it during the week. It true that I focused to functionality. Now I working at test hardware boards. One person is not enough for all my plans. If you want you could update it also. I will add it into source code.

**Comment by mamin27** [comment]
I was added documentation to separate Markdown files for each Perl, Free Pascal code. If you have any question don't hesitate to send me feedback.
